src/functions/raisebox.js

```diff
@@ -8,7 +8,32 @@ import {calculateSize} from "../units";
 import * as html from "../buildHTML";
 import * as mml from "../buildMathML";
 
-// Box manipulation
+// \raisebox, \raise, and \lower
+
+const htmlBuilder = (group, options) => {
+    const body = html.buildGroup(group.body, options);
+    let dy = calculateSize(group.dy, options);
+    if (group.funcName === "\\lower") {
+        dy *= -1;
+    }
```

Comment:
If we do the `\lower` check in the handler this can be simplified to:
```suggestion
    const dy = calculateSize(group.dy, options);
```